EnvironmentSimulator/Libraries/esminiLib/ESMiniWrapper.cs

```diff
@@ -236,6 +259,18 @@ public static class ESMiniLib
         /// <returns> @return Id [0..inf] of the added object successful, -1 on failure</returns>
         public static extern int SE_AddObject(string object_name, int object_type, int object_category, int object_role, int model_id);
 
+        [DllImport(LIB_NAME, EntryPoint = "SE_AddObjectWithBoundingBox")]
+        /// <summary>Add object with bounding box. Sets scale mode to MODEL_TO_BB. Should be followed by one of the SE_Report functions to establish initial state.</summary>
```

Comment:
Suggestion:
/// <summary>Add object with specified bounding box. Should be followed by one of the SE_Report functions to establish initial state.</summary>